src/views/partials/podcasts/create.hbs

```diff
@@ -0,0 +1,21 @@
+<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
+<script src="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.js"></script>
```

Comment:
These as well, no? 🙂

Reply:
True, I left the content section in at first to add descriptions to the videos. I decided against it in the end but must have forgotten to double check the rest of the file. Thanks for spotting it 👍